Frameworks/UIKit/UIViewController.mm

```diff
@@ -1233,105 +1253,119 @@ - (void)_addToTop:(NSNumber*)animatedValue {
 
     priv->_isRootView = true;
 
+    bool displayPopover = false;
```

Comment:
I've made the two s/bool/BOOL/ changes and squashed all review changes into a single review comments commit.